src/index.css

```diff
@@ -23,14 +23,9 @@ a:hover {
 }
 
 body {
-<<<<<<< Updated upstream
-  margin: 0;
```

Comment:
Are you sure you want to do this?

Reply:
@Jgarnes Can you respond to this comment?

Reply:
Yes I'm sure.